packages/docusaurus-1.x/lib/core/toc.js

```diff
@@ -27,16 +26,18 @@ function getTOC(content, headingTags = 'h2', subHeadingTags = 'h3') {
     ? [].concat(subHeadingTags).map(tagToLevel)
     : [];
   const allowedHeadingLevels = headingLevels.concat(subHeadingLevels);
-  const md = new Remarkable();
+  const md = new Remarkable({
+    // Enable HTML tags in source (same as './renderMarkdown.js')
+    html: true,
+  });
   const headings = mdToc(content).json;
   const toc = [];
   const slugger = new GithubSlugger();
   let current;
 
   headings.forEach(heading => {
     const rawContent = heading.content;
-    const safeContent = striptags(rawContent);
```

Comment:
We should find out when/why this was added. We could be regressing on a bug.

Reply:
It's because of https://github.com/facebook/docusaurus/pull/1762 which was a wrong solution.

So the reason why #2021 happens (inconsistent) is because in right TOC, we try to strip tags, and then render with remarkable.

In markdown itself, we only render with remarkable.

The remarkable render function is also different.
https://github.com/facebook/docusaurus/blob/91902ed709577ae2bd4c3f696519b662095d7f60/packages/docusaurus-1.x/lib/core/renderMarkdown.js#L86

vs https://github.com/facebook/docusaurus/blob/91902ed709577ae2bd4c3f696519b662095d7f60/packages/docusaurus-1.x/lib/core/toc.js#L30

Pretty sure this is 99.99% the right fix

Reply:
#1762 attempted to fix a bug whereby

```markdown
## Referencing Site Documents <span id="jump">
```

![image](https://user-images.githubusercontent.com/17883920/69305308-1b1fa180-0c56-11ea-8ffd-0550fb650343.png)


Should be rendered on the right without the `span` in TOC.

But its causing inconsistent content.

This PR on other hand fixed that bug but is still consistent